### DISCUSSION TITLE
Is there a way to run tox in parallel mode without executing the parallel sub command of the CLI?

### DISCUSSION THREAD

**Discussion by droctothorpe** [tox-dev/tox#3255] [Q&A] [answered]
Our CI is extremely locked down and there's no way to manipulate the tox execution command to leverage the `parallel` sub command. Is there anyway to tell tox to run in parallel mode through env vars or the configuration file? Thank you!

**Comment by jugmac00** [comment, marked as answer]
I am afraid I am not aware of any way to do so with env variables or with configuration only.

The only way I could imagine was to create a tox plugin which either makes this behaviour configurable or for simplicity forces parallel mode for all invocations.

**Reply by droctothorpe** [reply]
Thank you! A plugin would still require petitioning the CI powers that be, at which point I might as well just demand support for flags. Much appreciated!
